src/utils/input_bam.rs

```diff
@@ -21,12 +21,19 @@ pub struct FiberFilters {
         help_heading = "BAM-Options"
     )]
     pub bit_flag: u16,
+    /// Filtering expression to use for filtering records
```

Comment:
You can extend this by adding more lines starting with /// 

It would be great if you used it as a space to describe the syntax of the parser.

Reply:
See db994b82aec8293856e325d90f960d57fd3d8df8